doc/developer-guides/hld/hv-interrupt.rst

```diff
@@ -470,7 +470,7 @@ related operations.
    void free_irq(uint32_t irq)
       /*   Free irq num and unregister the irq action. */
 
-   void set_irq_trigger_mode(uint32_t irq, bool is_level_trigger)
+   void set_irq_trigger_mode(uint32_t irq, bool is_level_triggered)
```

Comment:
The mission here is to remove documentation of APIs written like this, and replace with documentation pulled from the API comments in the code, so this line (and all the other function declarations in the doc) should be instead:

    .. doxygenfunction:: set_irq_trigger_mode
       :project: Project ACRN

For other elements such as structs or unions you want to show up in the documentation, you'd use ``doxygenstruct`` or ``doxygenunion`` (as documented in https://breathe.readthedocs.io/en/latest/directives.html).

Reply:
Done in the last update.